lib/streamwrapper.php

```diff
@@ -702,6 +703,7 @@ private function triggerError($errors, $flags = null) {
 		if ($flags & STREAM_URL_STAT_QUIET) {
 			return $flags & STREAM_URL_STAT_LINK
 				// This is triggered for things like is_link()
+				/* @phan-suppress-next-line PhanTypeMismatchArgument, PhanTypeMismatchReturn */
```

Comment:
I suppressed this for now.
This is weird stuff - `triggerError` says that it returns `boolean` but `formatUrlStat` makes an array???
"someone" should work out what this code was supposed to be doing.